suite-native/alerts/src/alertsAtoms.ts

```diff
@@ -6,8 +6,16 @@ import { ButtonAccessory, ButtonColorScheme, PictogramVariant } from '@suite-nat
 import { IconName } from '@suite-native/icons';
 import { NativeSpacing } from '@trezor/theme';
 
+export type AlertType =
+    | 'autoEject'
+    | 'bluetoothPairing'
+    | 'bluetoothStatus'
```

Comment:
Please let's got with `bluetoothAdapter`. It's a bit more precise and would come before `bluetoothPairing`.

Reply:
https://github.com/trezor/trezor-suite/pull/22077/commits/271bbafe511f55adaa8245e6b8c0cc2afcb8c3f5